setup.cfg

```diff
@@ -18,14 +18,29 @@ version = 0.6
 [options]
 python_requires = >=3.5
 install_requires =
-    numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'
+    # NOTE: platform_system 'AIX' and platform_machine 'aarch64' are mutually
+    #       exclusive so they are specified by themself when setting their
+    #       numpy version.
```

Comment:
I'm fairly certain this is correct, but if someone knows better I'd definitely want their advice.

Reply:
Yes that's correct.